src/freertos_drivers/st/Stm32Railcom.hxx

```diff
@@ -24,57 +24,85 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  *
- * \file TivaRailcom.hxx
+ * \file Stm32Railcom.hxx
  *
- * Device driver for TivaWare to read one or more UART inputs for railcom data.
- *
- * usage:
- *
- * struct MyRailcomPins {
- *   // add all definitions from the example RailcomHw here
- * };
- * // Some definitions need to go outside of the class.
- * // no need for attribute weak if this is in a .cxx file.
- * const uint32_t MyRailcomPins::UART_BASE[] __attribute__((weak)) = {...};
- * ...
- *
- * TivaRailcomDriver<MyrailcomPins> railcom_driver("/dev/railcom");
- *
- * // assuming you put OS_INTERRRUPT = INT_UART3 into the structure.
- * void uart3_interrupt_handler() {
- *    railcom_driver.os_interrrupt();
- * }
- *
- * Then open /dev/railcom and read dcc::Feedback structures from it. Each read
- * much be exactly sizeof(dcc::Feedback) length. If there are no more packets to
- * read, you'll get return=0, errno==EAGAIN. Add an ioctl CAN_READ_ACTIVE to
- * get a notification when there is something to read.
+ * Device driver for STM32 chips to read one or more UART inputs for railcom
+ * data.
  *
  * @author Balazs Racz
- * @date 6 Jan 2015
+ * @date 6 Mar 2023
```

Comment:
Update the copyright year on line 2 above.

Reply:
done